components/avatar/avatar.vue

```diff
@@ -308,8 +324,8 @@ export default {
     },
 
     validateImageAttrsPresence () {
-      const isSrcMissing = !this.$refs.canvas.firstChild.getAttribute('src');
-      const isAltMissing = !this.$refs.canvas.firstChild.getAttribute('alt');
+      const isSrcMissing = !this.$slots.default[0].elm.getAttribute('src');
+      const isAltMissing = !this.$slots.default[0].elm.getAttribute('alt');
```

Comment:
Mhm... If we are using our local `defaultSlotContent` and also updating it in `updated()` hook... why we are accessing directly to the $slots here?, could we use `defaultSlotContent` ...? Or Im missing sth...

Just asking I didnt switch the branch

Reply:
Right, I missed that one, nice catch.